examples/widgets/check_buttons.py

```diff
@@ -32,7 +32,10 @@
 check = CheckButtons(
     ax=rax,
     labels=lines_by_label.keys(),
-    actives=[l.get_visible() for l in lines_by_label.values()]
+    actives=[l.get_visible() for l in lines_by_label.values()],
+    label_props=[{'color': 'k'}, {'color': 'r'}, {'color': 'g'}],
+    frame_props={'edgecolor': ['k', 'r', 'g']},
+    check_props={'facecolor': ['k', 'r', 'g']},
```

Comment:
I looked at [the rendered doc](https://output.circle-artifacts.com/output/job/c61a878a-ba86-4bc0-aced-85070efc5522/artifacts/0/doc/build/html/gallery/widgets/check_buttons.html#sphx-glr-gallery-widgets-check-buttons-py) and noticed that the Xs are still black, I think maybe `edgecolor` is what was intended here?
```suggestion
    check_props={'edgecolor': ['k', 'r', 'g']},
```

Reply:
No, the bug is in `widgets.py`; I just need to work out what exactly the semantics for color/facecolor/edgecolor will be.